2021-10-02 12:05:54,890 INFO    MainThread:40106 [wandb_setup.py:_flush():69] setting env: {}
2021-10-02 12:05:54,890 INFO    MainThread:40106 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-02 12:05:54,890 INFO    MainThread:40106 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20211002_120554-vxp7ulwz/logs/debug.log
2021-10-02 12:05:54,890 INFO    MainThread:40106 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20211002_120554-vxp7ulwz/logs/debug-internal.log
2021-10-02 12:05:54,890 INFO    MainThread:40106 [wandb_init.py:init():381] calling init triggers
2021-10-02 12:05:54,891 INFO    MainThread:40106 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-10-02 12:05:54,891 INFO    MainThread:40106 [wandb_init.py:init():430] starting backend
2021-10-02 12:05:54,891 INFO    MainThread:40106 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-02 12:05:54,897 INFO    MainThread:40106 [backend.py:ensure_launched():135] starting backend process...
2021-10-02 12:05:54,963 INFO    MainThread:40106 [backend.py:ensure_launched():140] started backend process with pid: 41190
2021-10-02 12:05:54,974 INFO    MainThread:40106 [wandb_init.py:init():435] backend started and connected
2021-10-02 12:05:54,978 INFO    MainThread:40106 [wandb_init.py:init():494] updated telemetry
2021-10-02 12:05:54,979 INFO    MainThread:40106 [wandb_init.py:init():517] communicating current version
2021-10-02 12:05:56,008 INFO    MainThread:40106 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-02 12:05:56,009 INFO    MainThread:40106 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-02 12:05:56,212 INFO    MainThread:40106 [wandb_init.py:init():557] starting run threads in backend
2021-10-02 12:06:01,221 INFO    MainThread:40106 [wandb_run.py:_console_start():1605] atexit reg
2021-10-02 12:06:01,222 INFO    MainThread:40106 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-10-02 12:06:01,222 INFO    MainThread:40106 [wandb_run.py:_redirect():1484] Redirecting console.
2021-10-02 12:06:01,226 INFO    MainThread:40106 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-02 12:06:01,226 INFO    MainThread:40106 [wandb_init.py:init():582] run started, returning control to user process
2021-10-02 12:06:33,003 INFO    MainThread:40106 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-10-02 12:06:33,004 INFO    MainThread:40106 [wandb_run.py:_restore():1547] restore
2021-10-02 12:06:35,809 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 12:06:36,249 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 12:06:36,384 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 12:06:37,274 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 25110
}

2021-10-02 12:06:37,376 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 25110
}

2021-10-02 12:06:37,479 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}

2021-10-02 12:06:37,582 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}

2021-10-02 12:06:37,685 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}

2021-10-02 12:06:37,787 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}

2021-10-02 12:06:37,889 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}

2021-10-02 12:06:38,093 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}

2021-10-02 12:06:38,370 INFO    MainThread:40106 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 25110
  total_bytes: 25110
}
local_info {
}

2021-10-02 12:06:39,702 INFO    MainThread:40106 [wandb_run.py:_show_files():1947] logging synced files
